third_party/bigframes_vendored/pandas/core/indexes/base.py

```diff
@@ -25,15 +25,180 @@ class Index:
     """
 
     @property
-    def name(self):
-        """Returns Index name."""
+    def names(self):
```

Comment:
I think name already has a docstring in your other PR: 
https://github.com/googleapis/python-bigquery-dataframes/pull/1144/files#diff-dcbf92e700ac22797e35504f1a75fc8469dc25049d7956b06aef910d454198b3R26

Reply:
Thanks for flagging this. I see that I added both `name` and `names` on the same line. I will consolidate the changes in one PR to avoid the merge conflict.